ui/selectors/transactions.js

```diff
@@ -56,6 +56,19 @@ export const getUnapprovedTransactions = createDeepEqualSelector(
   (transactions) => transactions,
 );
 
+export const getApprovedAndSignedTransactions = createDeepEqualSelector(
+  (state) => {
+    const currentNetworkTransactions = getCurrentNetworkTransactions(state);
+
+    return currentNetworkTransactions.filter((transaction) =>
+      [TransactionStatus.approved, TransactionStatus.signed].includes(
+        transaction.status,
+      ),
+    );
+  },
+  (transactions) => transactions,
+);
```

Comment:
I'm not totally up to speed with this particular code. But, from what I know about selectors, I was expecting it to look something like this. What do you think?
```ts
export const getApprovedAndSignedTransactions = createDeepEqualSelector(
  getCurrentNetworkTransactions,
  (currentNetworkTransactions) => {
    return currentNetworkTransactions.filter((transaction) =>
      [TransactionStatus.approved, TransactionStatus.signed].includes(
        transaction.status,
      ),
    );
  },
);
```

Reply:
This is an interesting point but I've matched the existing format of `getUnapprovedTransactions` for now for consistency.

We can test this out and potentially update both in a future PR.